pkg/util/steps/runner_test.go

```diff
@@ -379,8 +379,8 @@ func TestStepMetricsTopicNaming(t *testing.T) {
 		},
 	} {
 		t.Run(tt.desc, func(t *testing.T) {
-			if got := tt.step.metricsTopic(); got != tt.want {
-				t.Errorf("incorrect step metrics topic, want: %s, got: %s", tt.want, got)
+			if got := tt.step.metricsName(); got != tt.want {
+				t.Errorf("incorrect step metrics name, want: %s, got: %s", tt.want, got)
```

Comment:
can we rename this test from `TestStepMetricsTopicNaming` -> `TestStepMetricsNameFormatting` or similar?

Reply:
Yes! Aligned with the refactoring. Done! 
Thanks!